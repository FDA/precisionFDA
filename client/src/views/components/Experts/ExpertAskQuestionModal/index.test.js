import React from 'react'
import { shallow } from 'enzyme'

import { ExpertAskQuestionModal } from '.'


describe('ExpertAskQuestionModal test', () => {
  it('should render', () => {
    const component = shallow(<ExpertAskQuestionModal />)

    expect(component).toMatchSnapshot()
  })
})
