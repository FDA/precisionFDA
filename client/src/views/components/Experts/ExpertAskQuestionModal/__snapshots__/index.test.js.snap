// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpertAskQuestionModal test should render 1`] = `
<Component
  reCaptchaKey="some-key"
  useEnterprise={true}
>
  <div
    className="objects-actions-modal"
  >
    <Modal
      hideModalHandler={[Function]}
      modalFooterContent={
        <Footer
          action={[Function]}
          hideAction={[Function]}
          isAskingDisabled={true}
        />
      }
      noPadding={true}
      title="Submit a new question"
    >
      <div
        className="modal-header"
      >
        <span>
          Asking as:Â 
          Anonymous
        </span>
        <div
          className="modal-body"
        >
          <TextareaField
            aria_label="Ask a question to submit to Expert"
            label=""
            name="asking"
            onChange={[Function]}
            placeholder="Submit a question.."
            style={
              Object {
                "height": "100px",
              }
            }
            value=""
          />
        </div>
      </div>
    </Modal>
  </div>
</Component>
`;
